### PR TITLE
Add stretched link variations from plugin

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
No ticket needed.

# Description
[Stretched link variations styles](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/tree/9cd9a3cfe4f8b401b2fddd2fe40fcebb51610f0f/assets/src/variations/stretched-link) and [block variation in JS](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/e980c81583a3ffdf32ad86d718e0e6495caa1d9e/assets/src/editorIndex.js#L37-L50) are Required in master theme.

Also required by:
- https://github.com/greenpeace/planet4-master-theme/pull/2258
- https://github.com/greenpeace/planet4-master-theme/pull/2259

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [venus](https://www-dev.greenpeace.org/test-venus) | [admin](https://www-dev.greenpeace.org/test-venus/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-venus/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-venus) | [composer-local.json](https://github.com/greenpeace/planet4-test-venus/blob/main/composer-local.json)

:watch: 2024.04.24 15:56:01

**Review by mleray — CHANGES_REQUESTED** [review]
We also need to [register the block variation in JS](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/e980c81583a3ffdf32ad86d718e0e6495caa1d9e/assets/src/editorIndex.js#L37-L50)

File: assets/src/block-variations/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const {registerBlockVariation} = wp.blocks;
+const {__} = wp.i18n;
+
+/*
+ This file will register general block variations. In case of register a very
```
Comment (GP-Dan-Tovbein) [inline assets/src/block-variations/index.js:5]: Added this legend

**Review by mleray — APPROVED** [review]
Tested on the instance, looks good to me 👍

**Pull request merged**